src/components/tabs/tabs.ts

```diff
@@ -261,10 +266,34 @@ export class Tabs extends Ion implements AfterViewInit, RootNode, ITabs {
       viewCtrl._setContent(this);
       viewCtrl._setContentRef(elementRef);
     }
+
+    if (keyboard) {
```

Comment:
if the webview is not resizes this code does not really matter, read config.keyboardResizes?